### ISSUE TITLE
Aria-hidden true should not be applied to "Remove" cross icon 

### ISSUE THREAD

**Issue by Bojhan** [TheIdentitySelector/thiss-js#281]
We currently "hide" the remove SVG from screenreaders. 

By fixing #264, we will likely remove this overarching issue - that the styling and `hyperlink` is applied directly on the icon.

`<svg class="svg-inline--fa fa-times fa-w-11 institution-remove remove" tabindex="0" aria-labelledby="Remove Institute Central Washington University from remembered institutions" focusable="false" data-prefix="fa" data-icon="times" role="img" xmlns="http://www.w3.org/2000/svg" viewBox="0 0 352 512" data-fa-i2svg="" aria-hidden="true">`

After fixing #264 evaluate if this issue still exist, and remove the `aria-hidden` true mark. The `aria-labelby` should be more descriptive in that case, indicating its about the "icon" not the action.

**Comment by enriquepablo** [comment]
I don't see any `aria-hidden=true` there...

**Comment by Bojhan** [comment]
It was fixed in an earlier release, just not yet deployed to live.
